### ISSUE TITLE
Proto file with import causes error during REST call

### ISSUE THREAD

**Issue by aaronggi** [Kong/kong#7470]
It appears the file import works correctly, but during the protobuf encoding, there is an error that occurs. This only happens with a .proto file that imports another.

I have grpc-gateway with the latest, being loaded as a custom plugin for kong, for debugging purposes.

When I call the grpc service via grpcurl (through kong's http2 proxy listen port) it works fine. 

Here's my repo I've been running some testing with:
https://github.com/aaronggi/kong-project-poc

Not sure if there's something I'm missing here, but if you want to test it out, you can reproduce with the following:
`docker compose up --build -d &&\`
`curl -XGET http://localhost:8000/CommonTest1/SomeRandomString`

Error in Kong log:
``` 
kong_1           | 2020/11/19 20:53:33 [error] 28#0: *2952 lua coroutine: runtime error: /usr/local/custom/kong/plugins/grpc-gateway/deco.lua:253: bad argument Kong/kong-plugin-grpc-gateway#1 to 'encode' (type '.AIRproto.CommonReq' does not exists)
kong_1           | stack traceback:
kong_1           | coroutine 0:
kong_1           |      [C]: in function 'encode'
kong_1           |      /usr/local/custom/kong/plugins/grpc-gateway/deco.lua:253: in function 'upstream'
kong_1           |      /usr/local/custom/kong/plugins/grpc-gateway/handler.lua:56: in function </usr/local/custom/kong/plugins/grpc-gateway/handler.lua:36>
kong_1           | coroutine 1:
kong_1           |      [C]: in function 'resume'
kong_1           |      coroutine.wrap:21: in function <coroutine.wrap:21>
kong_1           |      /usr/local/share/lua/5.1/kong/init.lua:756: in function 'access'
kong_1           |      access_by_lua(nginx-kong.conf:88):2: in main chunk, client: 172.20.0.1, server: kong, request: "GET /CommonTest1/Aaron HTTP/1.1", host: "localhost:8000" 
kong_1           | 2020/11/19 20:53:33 [error] 28#0: *2952 [kong] init.lua:758 [grpc-gateway] /usr/local/custom/kong/plugins/grpc-gateway/deco.lua:253: bad argument Kong/kong-plugin-grpc-gateway#1 to 'encode' (type '.AIRproto.CommonReq' does not exists), client: 172.20.0.1, server: kong, request: "GET /CommonTest1/Aaron HTTP/1.1", host: "localhost:8000"
kong_1           | 172.20.0.1 - - [19/Nov/2020:20:53:33 +0000] "GET /CommonTest1/Aaron HTTP/1.1" 500 0 "-" "PostmanRuntime/7.26.8"
kong_1           | 2020/11/19 20:53:38 [debug] 28#0: *2968 [lua] init.lua:288: [cluster_events] polling events from: 1605818313.207
kong_1           | 2020/11/19 20:53:43 [debug] 28#0: *2984 [lua] init.lua:288: [cluster_events] polling events from: 1605818313.207
kong_1           | 2020/11/19 20:53:48 [debug] 28#0: *3000 [lua] init.lua:288: [cluster_events] polling events from: 1605818313.207
kong_1           | 2020/11/19 20:53:53 [debug] 28#0: *3016 [lua] init.lua:288: [cluster_events] polling events from: 1605818313.207
kong_1           | 2020/11/19 20:53:56 [warn] 28#0: *3022 a client request body is buffered to a temporary file /usr/local/kong/client_body_temp/0000000002, client: 172.20.0.1, 
server: kong, request: "POST /CommonTest1.Test1/RunTest1 HTTP/2.0", host: "localhost:9080"
kong_1           | 2020/11/19 20:53:56 [debug] 28#0: *3022 [lua] init.lua:915: balancer(): setting address (try 1): 172.20.0.5:5005
kong_1           | 2020/11/19 20:53:56 [debug] 28#0: *3022 [lua] init.lua:944: balancer(): enabled connection keepalive (pool=172.20.0.5|5005, pool_size=60, idle_timeout=60, max_requests=100)```

**Comment by git-torrent** [comment]
We fixed this in `get_proto_info()` function at https://github.com/Kong/kong/blob/master/kong/plugins/grpc-gateway/deco.lua#L84 by putting `p:loadfile(name)` from https://github.com/Kong/kong/blob/master/kong/plugins/grpc-gateway/deco.lua#L132 just before `p:parsefile(name)` at https://github.com/Kong/kong/blob/master/kong/plugins/grpc-gateway/deco.lua#L94

I can prepare PR for this.

**Comment by javierguerragiraldez** [comment]
fixed by #7538  (which incorporates the change submitted by #7563)
